postcode_lookup/templates/index.html

```diff
@@ -36,17 +35,20 @@ <h1 class="c-hero__title">
                                 Search
                             </button>
 
-
                         </div>
+                    <p class="dc-form-hint">eg. SW1A 2AA</p>
```

Comment:
This is really minor stuff, but..

I think we could do wit a tiny bit more vertical space above this. When the textbox is focussed the extra border kind of runs into the hint slightly. The previous hint style had a bit more vertical space from the text box.

Also, this is not actually new in this PR but while we're looking at it.. the same thing happens even more with the button.

<img width="633" height="173" alt="Screenshot at 2025-08-12 10-25-29" src="https://github.com/user-attachments/assets/31bf81b6-7c57-449b-98a0-3541ec58da58" />

Also, I was going to say "eg." --> "e.g", but we might want to do neither and write "for example" in full https://www.gov.uk/guidance/style-guide/a-to-z#eg-etc-and-ie

Reply:
<img width="696" height="213" alt="image" src="https://github.com/user-attachments/assets/41ab8769-793b-4d80-8e39-06be6e41406e" />

Good call, more space given and words changed. In terms of the focus outline: this is a problem with every other search box on their site at the moment, e.g https://www.electoralcommission.org.uk/voting-and-elections/register-vote#check-if-registered-en. I don't really know what to do here, but it feels like keeping in line with their design is better than trying to roll our own solution?